dev-python/shiboken/shiboken-9999.ebuild

```diff
@@ -1,38 +1,41 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
-PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
 
 inherit cmake-utils python-r1 git-r3
 
 DESCRIPTION="A tool for creating Python bindings for C++ libraries"
-HOMEPAGE="https://wiki.qt.io/Pyside"
+HOMEPAGE="https://wiki.qt.io/PySide2"
 EGIT_REPO_URI=(
-	"git://code.qt.io/pyside/${PN}.git"
-	"https://code.qt.io/git/pyside/${PN}.git"
+	"git://code.qt.io/pyside/shiboken.git"
+	"https://code.qt.io/git/pyside/shiboken.git"
 )
 
 LICENSE="LGPL-2.1"
-SLOT="0"
+SLOT="2"
 KEYWORDS=""
 IUSE="test"
-
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
+# Minimum Qt version required, While PySide2 requires specific Qt 5 releases,
+# Shiboken2 purports to support all Qt 5 releases.
+QT_PV="5:5="
```

Comment:
> It seems to follow that updating Qt[4|5] should silently break all currently installed versions of Shiboken[1|2] and hence PySide[1|2].

If shiboken only uses/wraps the public portion of the Qt API, then it should not break when updating Qt (Qt maintains backward API & ABI compatibility across minor versions). If shiboken does use some private Qt stuff, then that could be a problem.

This is a crucial point and definitely requires more investigation.